### DISCUSSION TITLE
Sidebar animation

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#758] [Ideas]
### What feature would you like to see?

I want to see sidebar animation when opening and closing, but there must be a delay of example 500ms. So when quickly moving the mouse over the sidebar, I don't want to see the fast open and close.

https://github.com/user-attachments/assets/9d3b3d02-b75c-459d-89cc-3218b8c8a18f

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/700_

**Comment by markox92** [comment]
I posted in the Issue page example how this can be done with CSS but this can make better with just few JavaScript line, you can have amazing sidebar bar animation there.
